universe/Species.cpp

```diff
@@ -30,6 +30,15 @@ FocusType::FocusType(std::string& name, std::string& description,
     m_graphic(std::move(graphic))
 {}
 
+FocusType::FocusType(std::string&& name, std::string&& description,
+                     std::unique_ptr<Condition::Condition>&& location,
+                     std::string&& graphic) :
```

Comment:
if the only difference is `string&&` instead of `string&`, check if you still need the `&` version after migrating the parsers?